README.md

```diff
@@ -2,22 +2,32 @@
 
 # OpenTracing Spring Web Instrumentation
 
-This library provides instrumentation for Spring  Web applications (Boot and MVC). It creates tracing data for 
-server requests and also client requests (`RestTemplate` and `AsyncRestTemplate`).
+This library provides instrumentation for Spring Web applications (Boot, MVC and WebFlux). It creates tracing data for 
+server requests and also client requests (`RestTemplate`, `AsyncRestTemplate` and `WebClient`).
 
 ## Use [opentracing-spring-cloud](https://github.com/opentracing-contrib/java-spring-cloud) instead
 
 As it was mentioned above this library traces only inbound/outbound HTTP requests. If you would like to 
 get automatically traced different set of technologies e.g. `spring-cloud-netflix`, JMS or even more then
 use project [opentracing-spring-cloud](https://github.com/opentracing-contrib/java-spring-cloud) instead.
 
+For reactive applications, it is especially recommended to use `reactor` tracing from
```

Comment:
Is reactor supported in opentracing-spring-cloud?

Reply:
I'm planning to add that in a separate pull request. I already have the code ready for it, but `opentracing-reactor` needs to be published to Maven central first. (See https://github.com/opentracing-contrib/java-reactor/issues/1)